### PR TITLE
Warmup Complete 1/26/24

### PR DESCRIPTION
**Pull request by tylermarcott**

### CODE REVIEW

File: Tyler/Challenge 1/26/style.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
+
+
+body{
+  background-color: rgb(59, 59, 59);
+}
+
+.trailer-button{
```
Comment (MichaelJHTaggart) [inline Tyler/Challenge 1/26/style.css:7]: This styling class is missing a :hover psuedo style. There is no hovering styling.

- [x] button looks like it does on website
- [ ] hovering over button has same styling change as website.

**Review by MichaelJHTaggart — CHANGES_REQUESTED** [review]
Just update with the hover styles and you are all set!

File: Tyler/Challenge 1/26/style.css (hunk 1 above)
Comment (tylermarcott) [reply Tyler/Challenge 1/26/style.css:7]: Added the on hover.

**Review by MichaelJHTaggart — APPROVED** [review]
Nice! Looks good!

**Pull request merged**